web/src/app/profile/components/Profile.tsx

```diff
@@ -23,9 +23,7 @@ export default function Profile() {
           </p>
 
           <p>
-            <Link href="/profile/approved-researcher" className="btn--form">
```

Comment:
css didn't seem to exist(?)

Reply:
@acholyn I guess you will be cleaning up the css as part of your upcoming PR? There are currently some unused classes in a few different files.